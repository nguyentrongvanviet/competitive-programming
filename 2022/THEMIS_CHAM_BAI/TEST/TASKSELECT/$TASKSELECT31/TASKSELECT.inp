10
6 398278678
3 263484156
6 220146351
2 301512107
7 672556405
4 163624467
1 134107869
1 633258475
7 971240702
8 1000000000
