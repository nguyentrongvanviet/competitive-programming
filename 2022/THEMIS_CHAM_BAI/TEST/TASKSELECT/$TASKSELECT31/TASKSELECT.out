522652829
866346683
543612688
150756053
482480115
497968957
1
1
829558506
794293046
